apps/hexagon_matmul/Makefile

```diff
@@ -0,0 +1,55 @@
+include ../support/Makefile.inc
```

Comment:
Those don't support running on Android. I think that maybe it's just all the Android running apps that need refactoring.